example/data/sample_workflow_view.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
we have the content of this file in readme, is it necessary to have it also here too?